packages/single-spa-welcome/src/welcome.js

```diff
@@ -1,7 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import "./welcome.css";
 
 export default function Root(props) {
+  const isUsingSingleSpaLayout = useState(() => {
+    // Search the dom for a single-spa-router element
+    // While this won't work for SSR, the goal of this app is to help
+    // beginners, which likely won't use SSR before running their
+    // first root-config.
+    const templates = Array.prototype.slice.call(
+      document.querySelectorAll("template")
+    );
+
+    return templates.some((template) => {
+      const hasSingleSpaRouter = !!template.content.querySelector(
```

Comment:
I don't think that `template.content` is defined in IE11, although this welcome parcel probably is fine without working in IE11.

Reply:
Oh poop. Yea, I don't find IE11 to be high priority, but then again, that's why I'm using Array.prototype.slice above instead of Array.from().